release-notes/opensearch-dashboards-reporting.release-notes-2.11.1.0.md

```diff
@@ -0,0 +1,12 @@
+## Version 2.11.1 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.11.0
+
+### Features
+Use core navigation instead of hard coding URLs ([#229](https://github.com/opensearch-project/dashboards-reporting/pull/229)) ([#231](https://github.com/opensearch-project/dashboards-reporting/pull/231))
```

Comment:
Feature? In a PATCH release?

https://semver.org/#spec-item-6
> Patch version Z (x.y.Z | x > 0) MUST be incremented if only backward compatible bug fixes are introduced. A bug fix is defined as an internal change that fixes incorrect behavior.

Have not all non-bugfix backports been revert?